run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+if [ $(command -v getprop) ]; then
+	echo detected android host, reading libraries from $PWD/lib
+	export LD_LIBRARY_PATH=$PWD/lib:$LD_LIBRARY_PATH
+fi
+
+exec ./bin/viamrtsp-android-arm64 $@
```

Comment:
is this always supposed to say android?

Reply:
No, `run.sh` entry is only used for Android right now. I have been using the appimage builds for testing on Linux.

Working on a patch that will use run.sh for linux as well - will test out and update PR with unified entry.

Reply:
Ok, updated so that Linux/Android module targets both work from `run.sh` entry.